README.md

```diff
@@ -9,14 +9,14 @@
 
 [**Quick Start**](https://llama-stack.readthedocs.io/en/latest/getting_started/index.html) | [**Documentation**](https://llama-stack.readthedocs.io/en/latest/index.html) | [**Colab Notebook**](./docs/getting_started.ipynb)
 
-
 ### ✨🎉 Llama 4 Support  🎉✨
 We released [Version 0.2.0](https://github.com/meta-llama/llama-stack/releases/tag/v0.2.0) with support for the Llama 4 herd of models released by Meta.
 
-You can now run Llama 4 models on Llama Stack.
+<details>
 
-*Note you need 8xH100 GPU-host to run these models*
+<summary>You can now run Llama 4 models on Llama Stack (click for details)</summary>
 
+*Note you need 8xH100 GPU-host to run these models*
```

Comment:
There needs to be a newline here or after opening up there is little space between summary and other content.

Also can we make click for details somehow a but more click-y (underlined), etc.? the little arrow on the left can be easily ignored.

Reply:
Added a newline. 

Also added a :fast_forward: to make this more obvious but happy to hear better options.

Reply:
fwiw in other parts of the documentation we use this occassionally:

https://github.com/meta-llama/llama-stack/blob/51492bd9b6d0f7342677b29c53629dd23d53b027/docs/source/getting_started/detailed_tutorial.md?plain=1#L316

Reply:
I switched to the same text. `{dropdown}` is not available for GitHub markdown renderer though so I stayed with the current syntax.